src/utils/storage/index.ts

```diff
@@ -17,15 +17,17 @@ enum StorageKeysEnum {
   ProfileData = 'profiledata',
   OtherProfileData = 'otherprofiledata',
   LastPromptNotification = 'lastPromptNotification',
-  LastSelectedMenu = 'lastSelectedMenu'
+  LastSelectedMenu = 'lastSelectedMenu',
+  BlockedStatus = 'blockedStatus',
+  BlockingStatus = 'blockingStatus'
 }
 
 interface IStorage {
-  get: () => string;
+  get: () => string | undefined;
   set: (value: string) => void;
   clear: () => void;
   setForKey: (key: string, value: string) => void;
-  getForKey: (key: string) => string;
+  getForKey: (key: string) => string | undefined;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in return type for `get` and `getForKey` methods from `string` to `string | undefined` is a good practice as it allows for better error handling when the key does not exist in storage. This change should be reflected wherever these methods are used to handle the possibility of an `undefined` return value.

```diff
-  get: () => string;
+  get: () => string | undefined;
   set: (value: string) => void;
   clear: () => void;
   setForKey: (key: string, value: string) => void;
-  getForKey: (key: string) => string;
+  getForKey: (key: string) => string | undefined;
   clearForKey: (key: string) => void;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->